xAPI.md

```diff
@@ -260,49 +260,30 @@ learning that SCORM could not enable.
 	*	[Appendix G: Cross Domain Request Example](#AppendixG)  
 
 <a name="partone"/>
-#Part One: About The Experience API
+#PART ONE: LEARNING EXPERIENCES
 
+<a name="Introduction"/>
```

Comment:
+1 to a convention. I like my way better than your way. <-(This is probably the heart of all standardisation problems.)